test/plugins/sqlite.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const expect = require("chai").expect;
+const Msg = require("../../src/models/msg");
+const Helper = require("../../src/helper");
+const MessageStorage = require("../../src/plugins/sqlite.js");
+
+describe("SQLite Message Storage", function() {
+	const expectedPath = path.join(Helper.getHomePath(), "logs", "testUser.sqlite3");
+	let store;
+
+	// Delete database file from previous test run
+	before((done) => {
```

Comment:
[Mocha recommends](https://mochajs.org/#arrow-functions) against using arrow functions, and sticking to regular functions ([`eslint-plugin-mocha` has `no-mocha-arrows`](https://github.com/lo1tuma/eslint-plugin-mocha/blob/master/docs/rules/no-mocha-arrows.md) but it can't be used along with `prefer-callback` yet), mind changing the different `() => {}`s into `function() {}`s?

Reply:
The only reason they discourage it is due to:
> If you do not need to use Mocha’s context, lambdas should work.

Which we are aware of and take into consideration.

Reply:
They also specify

> However, the result will be more difficult to refactor if the need eventually arises.

I would like us to use `eslint-plugin-mocha` at some point, this is just forward-thinking 🤷‍♂️